jvm-packages/xgboost4j/src/main/scala/ml/dmlc/xgboost4j/scala/Booster.scala

```diff
@@ -187,16 +187,43 @@ class Booster private[xgboost4j](private[xgboost4j] var booster: JBooster)
     booster.getModelDump(featureMap, withStats, format)
   }
 
+  /**
+    * Dump model as Array of string with specified feature names.
+    *
+    * @param featureNames Names of features.
+    */
+  @throws(classOf[XGBoostError])
+  def getModelDump(featureNames: Array[String])
+  : Array[String] = {
+    booster.getModelDump(featureNames, false, "text")
+  }
+
+  def getModelDump(featureNames: Array[String], withStats: Boolean, format: String)
+  : Array[String] = {
```

Comment:
indentation should follow https://github.com/databricks/scala-style-guide#indent

Reply:
Done.